RAM model overview page (#886)
ACCESS-NRI/access-hive.org.au: docs/models/configurations/index.md

            </span>
        </div>
    </a>

>     <!--
>     Need to update AM image with RAM figure
>     --> 

Reviewer: ```suggestion
```

    <a href="/models/configurations/access-ram/" class="horizontal-card">
        <div class="card-image-container">

>             <img src="/assets/model-config-logos/configurations-without-titles/access-am.png" class="white-background img-contain"></img> 

Reviewer: ```suggestion
            <img src="/assets/model-config-logos/configurations-without-titles/access-ram.png" class="white-background img-contain"></img> 
```
This will automatically work whenever the image is ready.

        </div>
        <div class="card-text-container">
            <span class="bold" >ACCESS-RAM</span>
            <span>

>                 ACCESS-RAM is an implementation of the UM regional nesting suite that uses ERA5 for lateral boundary driving conditions and initial conditions, with an option to replace land-surface with ERA5-land or BARRA-R2.

Reviewer: ```suggestion
                ACCESS-RAM is an implementation of the UKMO regional nesting suite that allows high-resolution regional atmosphere modelling.
```
